chef.gemspec

```diff
@@ -52,6 +52,7 @@ Gem::Specification.new do |s|
   s.add_dependency "addressable"
   s.add_dependency "syslog-logger", "~> 1.6"
   s.add_dependency "uuidtools", ">= 2.1.5", "< 3.0" # osx_profile resource
+  s.add_dependency "corefoundation", "~> 0.3.4" # corefoundation framework utilities
```

Comment:
the comment should say where we're using it more than what it gives us. 

```
# macos_userdefaults
```

Reply:
Done.